dvc/repo/fetch.py

```diff
@@ -72,7 +72,7 @@ def _fetch(
     if failed:
         raise DownloadError(failed)
 
-    return downloaded
+    return downloaded + len(used_run_cache)
```

Comment:
`fetch` and `push/pull` for run-cache did not add up counts. Was this intentional, @efiop?

Reply:
@skshetry to be honest, `--run-cache` was introduced asap as a hack to cover a specific use case, so I didn't add it to the count, hoping that we might find a better way (e.g. separate command). But it makes total sense to add it up, so let's keep it. :+1:

Reply:
@efiop, as far as I understand, `downloaded` should already have a total count, right? As we pass `used_run_cache` already.

Reply:
@skshetry Correct. So indeed `+ len(used_run_cache)` is not needed here for now.

Reply:
But, that's not working at all. See the tests.

Reply:
@skshetry Ok, the confusion here is because run cache consists of .dvc/cache/runs and the regular cache files assotiated with it. So `used_run_cache` is a list of run cache entries and regular `downloaded` accounts for regular cache files that include the ones used by used_run_cache. So `+ len(used_run_cache)` is correct if we want to account for run-cache entries and not only for the regular cache entries.

Reply:
Ahh, okay I got it. It's already transfer via `RunCache`. Thanks for the explanation.